### ISSUE TITLE
gridss_extract_overlapping_fragments.sh issue

### ISSUE THREAD

**Issue by katherinef** [PapenfussLab/gridss#443]
Hello,

I am trying to use the gridss_extract_overlapping_fragments.sh script and so I went through the instructions to build gridsstools from source.  However, when I try to run the gridss_extract_overlapping_fragments.sh script it asks me to specify the location of a jar file.  However, there is no gridss jar file in the installation.

Many thanks,

Kath

**Comment by d-cameron** [comment]
`gridsstools` is C code that requires building and recompilation if it does not run on your system (e.g. you're using a mac or a flavour of linux the precompiled binary does not run on). The GRIDSS jar is java code so does not require any recompilation - the binary .jar files included in GRIDSS releases work on all systems.

See the https://github.com/PapenfussLab/gridss#building-gridss section of the README.


TLDR: the jar is here: https://github.com/PapenfussLab/gridss/releases
